app/assets/stylesheets/provider/_links.scss

```diff
@@ -45,18 +47,18 @@ input[type='submit'].action {
     &:hover {
       color: $link-hover-color;
     }
-    // https://github.com/bokmann/font-awesome-rails/blob/master/vendor/assets/stylesheets/font-awesome.css.erb
-    // http://fortawesome.github.com/Font-Awesome/
-    &.undo:before, &.revert:before { @include iconic("\f0e2\0000a0"); }
+    // For reference:
+    // https://github.com/FortAwesome/font-awesome-sass/blob/5.15.1/assets/stylesheets/font-awesome/_variables.scss#L1319
+    &.undo:before, &.revert:before { @include iconic("\f2ea\0000a0"); }
```

Comment:
Original: https://fontawesome.com/v4/icon/undo

New: https://fontawesome.com/v5/icons/undo-alt?f=classic&s=solid

Optinally, we can update to a slightly different https://fontawesome.com/v5/icons/undo?f=classic&s=solid (in case this is what PF4 uses)